leaf-ui/js/onFunctions.js

```diff
@@ -794,13 +798,19 @@ paper.on("link:options", function (cell) {
         for (let element of graph.getElements()) {
             var cell = element.findView(paper).model;
             var intention = cell.get('intention');
-            var initSatVal = intention.getUserEvaluationBBM(0).get('assignedEvidencePair');
-            var funcType = intention.get('evolvingFunction').get('type');
+            
+            if (intention != null) {
+                var initSatVal = intention.getUserEvaluationBBM(0).get('assignedEvidencePair');
+                if (intention.get('evolvingFunction') != null) {
+                    var funcType = intention.get('evolvingFunction').get('type');
+                }
+            }
 
             // If the initsatVal is not empty and if funcType empty
             if (intention != null && initSatVal != '(no value)' && funcType === 'NT') {
                 intention.removeInitialSatValue();
                 cell.attr(".satvalue/text", "");
+                $('#init-sat-value').val('(no value)');
```

Comment:
Heads up that this changes to read '(no value)' even if the user has selected a node with an evolving function type (whose initial sat value doesn't clear). However, the user can click the intention again to refresh the inspector to show the correct initial sat value, so it's probably not a big issue.

Reply:
To recreate, add a node with an initial satisfaction value but no function, then add another node with an initial sat value + an evolving function, then clear evaluation labels. (Evaluation labels are only cleared if there is at least one node available on which to clear labels.)